token-group/interface/src/error.rs

```diff
@@ -1,13 +1,18 @@
 //! Interface error types
 
-use spl_program_error::*;
+use {
+    solana_decode_error::DecodeError,
+    solana_msg::msg,
+    solana_program_error::{PrintProgramError, ProgramError},
+};
 
 /// Errors that may be returned by the interface.
-#[spl_program_error(hash_error_code_start = 3_406_457_176)]
+#[repr(u32)]
+#[derive(Clone, Debug, Eq, thiserror::Error, num_derive::FromPrimitive, PartialEq)]
```

Comment:
Why not just update the macro?

Reply:
I tried, but that macro is quite tied to the assumption that there is a single solana_program crate that we import, and I didn't find a way to update it without breaking everything

Reply:
This is my fault, remember #7112?